src/app/@ansyn/ansyn/app-providers/overlay-source-providers/multiple-source-provider.ts

```diff
@@ -130,6 +130,7 @@ export class MultipleOverlaysSourceProvider extends BaseOverlaySourceProvider {
 
 	public fetch(fetchParams: IFetchParams): Observable<OverlaysFetchData> {
 		const mergedSortedOverlays: Observable<OverlaysFetchData> = Observable.forkJoin(this.sourceConfigs
+			.filter(s => !Boolean(fetchParams.dataInputFilters) ? true : fetchParams.dataInputFilters.some((dataInputFilter) => dataInputFilter.providerName === s.provider.sourceType))
```

Comment:
All pre-filtered sensors are passed here to all providers, will be fixed in a followup PR.